src/Data/Parity/HomomorphismNatParity.agda

```diff
@@ -0,0 +1,103 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some properties about parities
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Parity.HomomorphismNatParity where
+
+open import Data.Nat.Base
+  using (zero; suc; _+_; _*_; parity
+  ; +-rawMagma; +-0-rawMonoid
+  ; +-*-rawNearSemiring; +-*-rawSemiring)
+open import Data.Parity.Base using (Parity; 0ℙ; 1ℙ; _⁻¹)
```

Comment:
The style we're moving towards in the standard library is to import the module `as ℙ` and then write `ℙ.+` rather than rename to `_+ℙ_`.

Reply:
Thanks for the style hint. Fixed now, with 'symmetric' imports for `Nat` and `Parity` in that style, and revised import of `Sign` in the same spirit.